test/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -544,6 +544,89 @@ TEST(nntrainer_capi_nnmodel, addLayer_05_n) {
   EXPECT_EQ(status, ML_ERROR_NONE);
 }
 
+/**
+ * @brief Neural Network Model Get Layer Test
+ */
+TEST(nntrainer_capi_nnmodel, getLayer_01_p) {
+  int status = ML_ERROR_NONE;
+
+  ml_train_model_h model;
+  ml_train_layer_h add_layer;
+  ml_train_layer_h get_layer;
+
+  status = ml_train_model_construct(&model);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+
+  status = ml_train_layer_create(&add_layer, ML_TRAIN_LAYER_TYPE_INPUT);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+
+  status = ml_train_layer_set_property(add_layer, "name=inputlayer", NULL);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+
+  status = ml_train_model_add_layer(model, add_layer);
+  EXPECT_EQ(status, ML_ERROR_NONE);
+
+  status = ml_train_model_get_layer(model, "inputlayer", &get_layer);
+  EXPECT_EQ(status, ML_ERROR_NONE);
```

Comment:
Can you test the validity of the get_layer?

Reply:
One of the way is that you could set the input dim of input layer when you create and check the dim after you call the get_layer.

Reply:
I check it by set name of layer which is access by get layer API.

As @jijoongmoon mention in issue #1875 this should be replace by getProperty API which will be implement.